src/components/Navbar/index.jsx

```diff
@@ -46,8 +67,13 @@ export default function SiteNavbar() {
         </Form>
 
         <Button className="navButtons">{t("navbarsection.Search")}</Button>
-        <Button className="navButtons">{t("navbarsection.LogIn")}</Button>
-        <Button className="navButtons signUpButton">
```

Comment:
What about doing this here ` {currentUser ? (
           <div className="loggedInUserInfo">
             Logged in as {currentUser.name}
           </div>  <Logout />
         ):   <Button onClick={handleClickLogin} className="navButtons">
           {t("navbarsection.LogIn")}
         </Button>
         <Button
           onClick={handleClickSignup}
           className="navButtons signUpLogoutButton"
         >
           {t("navbarsection.SignUp")}
         </Button>}
        } `

Reply:
Yeah I can handle the buttons, but showing user name I need to think about it cuz I dont have an idea right now

Reply:
PTAL